src/components/App/App.js

```diff
@@ -1,13 +1,29 @@
-import "./App.css";
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <p>Backyard Burbs</p>
-      </header>
-    </div>
-  );
+import React, { Component } from 'react';
+import USAMap from "react-usa-map";
+import './App.css';
+
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      
+    }
+  }
+  mapHandler = (event) => {
+    alert(event.target.dataset.name);
```

Comment:
// let chosenState = target.dataset.name 
// call birdListFetch with the argument of chosenState - use that arguement to interpolate the getch call url 
// interpolate url with chosenState